src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -95,6 +77,18 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
         return null;
     }
 
+    public Boolean getCanDelete() {
```

Comment:
I liked the property name `canDelete` rather than `getCanDelete` which makes its usages more readable, but that is probably me just being finicky!

Reply:
We've got this getter bound in visualforce, so we need the `get` prefix here unless we go to the member-variable style getter/setters.

Reply:
👍🏽